maze_transformer/training/config.py

```diff
@@ -89,6 +90,13 @@ class TrainConfig:
         d_head=16,
         n_layers=4,
     ),
+    BaseGPTConfig(
```

Comment:
I would add a comment explaining how d_model, d_head, n_layers are calculated and possibly remove the math and add the final number